gamefixes-steam/409720.py

```diff
@@ -4,12 +4,7 @@
 
 
 def main() -> None:
-    """Disable ESYNC, disable intro's"""
-    # After loading the game, or a save file, a key needs to be pressed
-    # to continue. That screen does not respond to keyboard or mouse,
-    # so there is no way to continue. -nointro disables that screen
-    # (but also the intro's at the start of the game).
-    util.append_argument('-nointro')
-
-    # ESYNC causes texture problems and frequent hangs.
+    """Disable ESYNC and FSYNC"""
+    # ESYNC and FSYNC causes low quality texture problems in all BioShock 1 and 2 versions (Original and Remastered).
     util.disable_esync()
+    util.disable_fsync()
```

Comment:
Putting aside the user reports, which may be outdated at this time, it’s not clear to me where the fsync and esync issue was addressed for these titles. I’m not seeing it in the proton script nor in Valve’s fork of wine. Are you able to reference where exactly the issue has been addressed.

Reply:
To be clear, I’m referring to the commen your sourced:

https://github.com/ValveSoftware/Proton/issues/482#issuecomment-988954117

Reply:
In fact, if we’re to trust Kisak’s word, we can remove disabling fsync and esync all together. Adding these fixes in implies that this issue still exist and they’re wrong. If so, then this needs to be reported.

Reply:
I will do some further tests to see if these changes are necessary.

Reply:
I've gone ahead and put together a new PR #176 which addresses the necessary fixes.

If that's all good I'll go ahead and close this one.